docs/Reference/genesis.md

```diff
@@ -1,48 +1,92 @@
-      "balance": "1000000000000000000000000000"
-    },
-    "0xca843569e3427144cead5e4d5999a3d0ccf92b8e": {
-      "balance": "1000000000000000000000000000"
-    },
-    "0x0fbdc686b912d7722dc86510934589e0aaf3b55a": {
-      "balance": "1000000000000000000000000000"
-    },
-    "0x9186eb3d20cbd1f5f992a950d808c4495153abd5": {
-      "balance": "1000000000000000000000000000"
-    },
-    "0x0638e1574728b6d862dd5d3a3e0942c3be47d996": {
-      "balance": "1000000000000000000000000000"
+---
+description: Configuration items specified in the Hyperledger Besu genesis file
+---
+
+# Genesis file items
+
+The [GoQuorum genesis file](../HowTo/Configure/GenesisOptions.md) contains [network configuration items](#configuration-items)
+and [genesis block parameters](#genesis-block-parameters).
+
+## Configuration items
+
+Network configuration items are specified in the genesis file in the `config` object.
+
+| Item                | Description                                                                                                                                                                                 |
+|---------------------|-:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Milestone blocks    | [Milestone blocks for the network](#milestone-blocks).                                                                                                                                      |
+| `chainId`           | [Chain ID for the network](../Concepts/NetworkAndChainID.md).                                                                                                                               |
+| `istanbul`          | Specifies network uses [IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md) or [QBFT](../HowTo/Configure/Consensus-Protocols/QBFT.md) and contains [IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file) or [QBFT configuration items](../HowTo/Configure/Consensus-Protocols/QBFT.md#genesis-file), respectively. |
+| `clique`            | Specifies network uses [Clique](../HowTo/Configure/Consensus-Protocols/Clique.md) and contains Clique configuration items.                                                                  |
+| `txnSizeLimit`      | Maximum transaction size. The default is `64` (kilobytes), increased from Ethereum's default `32`. This is configurable up to `128`.                                                        |
+| `maxCodeSize`       | Maximum smart contract code size. The default is `32` (kilobytes), increased from Ethereum's default `24`. This is configurable up to `128`.                                                |
+| `isQuorum`          | Set to `true` to allow interoperable private transactions between GoQuorum and [Hyperledger Besu](https://besu.hyperledger.org/en/stable/) using the Tessera private transaction manager.   |
```

Comment:
`isQuorum` is more than that. it actually enables `geth` to work as GoQuorum and do some checks like make sure the gas fee is zero. for ex, you can use `isQuorum` to use the GoQuorum engine without using Tessera.